v3/docs/03-runtime/b-macros/index.mdx

```diff
@@ -192,11 +191,13 @@ pub struct Pallet<T>(_);
 
 **When to use**
 
-Highly encouraged, but optional which will help ensure all of your pallet storage items are bounded.
+You should not need to explicitly add this attribute.
```

Comment:
The macro got removed completely, since it would be a NO-OP now.  
Maybe worth mentioning.